base_classes/NXoptical_window.nxdl.xml

```diff
@@ -0,0 +1,132 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<?xml-stylesheet type="text/xsl" href="nxdlformat.xsl"?>
+<!--
+# NeXus - Neutron and X-ray Common Data Format
+#
+# Copyright (C) 2024-2025 NeXus International Advisory Committee (NIAC)
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+#
+# For further information, see http://www.nexusformat.org
+-->
+<!--
+A draft of a new base class to describe a window in a optical setup or device-->
+<definition xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" category="base" type="group" name="NXoptical_window" extends="NXaperture" xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd">
+    <doc>
+        A window of a cryostat, heater, vacuum chamber or a simple glass slide.
+        
+        This first version originates from NXoptical_spectroscopy to describe
+        cryostate windows and possible influences for ellipsometry measurements.
+        
+        For environmental measurements, the environment (liquid, vapor
+        etc.) is enclosed in a cell, which has windows both in the
+        direction of the source (entry window) and the detector (exit
+        window) (looking from the sample).
+        
+        The windows also add a phase shift to the light altering the
+        measured signal. This shift has to be corrected based on measuring
+        a known sample (reference sample) or the actual sample of interest
+        in the environmental cell. State if a window correction has been
+        performed in 'window_effects_corrected'. Reference measurements should be
+        considered as a separate experiment (with a separate NeXus file), and
+        the reference data shall be :ref:`linked &lt;Design-Links&gt;` in
+        ``reference_data_link``.
+        
+        The window is considered to be a part of the sample stage but also
+        beam path. Hence, its position within the beam path should be
+        defined by the 'depends_on' field.
+    </doc>
+    <group type="NXentry">
```

Comment:
This is unusual: current practise is to have `NXentry` groups at the root level only

Reply:
I checked if the implementation in the classes and appdefs that was voted upon resolved this comment - it does.